python/lsst/dbb/buffmngrs/handoff/local.py

```diff
@@ -194,3 +201,5 @@ def run(self):
                     logger.warning(f"Cannot remove '{path}': {ex}")
                 else:
                     logger.debug(f"Directory '{path}' removed successfully.")
+                    
+                
```

Comment:
Remove these two blank lines. Make user though that the last line ends with newline character though.

Reply:
I think you need to turn on the standard lint github action so that these are all spotted early.